lib/Elastica/Query.php

```diff
@@ -194,51 +171,45 @@ public function setHighlight(array $highlightArgs)
      *
      * @param mixed $highlight Add highlight argument
      *
-     * @return $this
-     *
      * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-highlighting.html
      */
-    public function addHighlight($highlight)
+    public function addHighlight($highlight): self
     {
         return $this->addParam('highlight', $highlight);
     }
 
     /**
      * Sets maximum number of results for this query.
      *
-     * @param int $size OPTIONAL Maximal number of results for query (default = 10)
+     * @param int $size Maximal number of results for query (default = 10)
      *
-     * @return $this
```

Comment:
Why do you consider the `return $this` as not needed anymore?

Reply:
The method signature is `method(): self`, the annotation is no superfluous.
I am planning to take all of those into consideration in a next PR (php-cs-fixer can help here ;-) )